web-runtime/src/faas/json/json_to_ivalues.rs

```diff
@@ -0,0 +1,244 @@
+/*
```

Comment:
Most of the code in marine-web is a direct copy-paste form runtime and faas, most of the copy-paste is cut to fit only marine-web purposes. Decoupling to a separate crate is the right thing, but i thing to move into the separate crates most of the `marine-runtime` and `fluence-faas` and make them dependent on a set of traits. If it will be done, decoupling only `fluence-faas/json` will be temporary thing. Do we still need it decoupled?

Reply:
Ok, this plan sounds reasonable